scenarios/rds_backup_monitoring/settings.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    opentelekomcloud = {
+      source  = "opentelekomcloud/opentelekomcloud"
+      version = ">= 1.22.0"
+    }
+  }
+  backend "s3" {
+    key                         = "terraform_state/rds_backup_monitoring"
+    endpoint                    = "obs.eu-de.otc.t-systems.com"
+    bucket                      = "obs-csm"
+    region                      = "eu-de"
+    skip_region_validation      = true
+    skip_credentials_validation = true
+  }
+}
+
+# Configure the OpenTelekomCloud Provider
+provider "opentelekomcloud" {
+  cloud = "devstack"
```

Comment:
To make testing easier, please pass the cloud name as `OS_CLOUD` env var instead of provider argument.